frontend/tests/e2e.sh

```diff
@@ -10,7 +10,15 @@ CYPRESS="${POD_NAME}-cypress"
 CONTAINERS+=( "${CYPRESS}" )
 
 podman build -f ${FRONTEND}/e2e_frontend.containerfile --tag cypress "${FRONTEND}"
-podman run ${POD} --name="${CYPRESS}" localhost/cypress
+
+mkdir -p "${FRONTEND}/coverage"
+
+# podman run --rm \
+#   -v "${FRONTEND}/coverage:/app/coverage" \
+#   --name="${CYPRESS}" \
+#   localhost/cypress 
```

Comment:
We don't need the old commented command. (Although the new line is rather long.)

Reply:
Updated it!